deploy-grafana.sh

```diff
@@ -1,5 +1,8 @@
 #!/usr/bin/env bash
-set -e
+
+set -euf -o pipefail
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
```

Comment:
set -euf and the DIR change here should be undone. The goal of these scripts is to be super simple and make it clear how one would deploy all of these Sourcegraph containers in their own setup -- while these changes may be better in general for bash, in this case they distract I think from the intent here. In other words, `set -e` and `$(pwd)` is good enough in this repository.

Reply:
done